examples/bone_finetuning/finetune.py

```diff
@@ -0,0 +1,371 @@
+import copy
+import logging
+import os
+import random
+from dataclasses import dataclass, field
+from typing import Dict, List, Literal, Optional, Sequence
```

Comment:
Not a big deal, but since we now dropped Python 3.9, you can use `dict` and `list` directly for type annotations, no need for `Dict` and `List` anymore.